testing/tests/index.test.ts

```diff
@@ -1613,19 +1613,17 @@ test("translated content broken links can fall back to en-us", () => {
   const { doc } = JSON.parse(fs.readFileSync(jsonFile, "utf-8")) as {
     doc: Doc;
   };
-  const map = new Map(doc.flaws.broken_links.map((x) => [x.href, x]));
-  expect(map.get("/fr/docs/Web/CSS/dumber").explanation).toBe(
-    "Can use the English (en-US) link as a fallback"
-  );
-  expect(map.get("/fr/docs/Web/CSS/number").explanation).toBe(
-    "Can use the English (en-US) link as a fallback"
-  );
+
+  const brokenHrefs = doc.flaws.broken_links.map((x) => x.href);
+  expect(brokenHrefs).not.toContain("/fr/docs/Web/CSS/dumber");
```

Comment:
This link is broken, and it should be redirected to `/fr/docs/Web/CSS/number`:

https://github.com/mdn/yari/blob/24a2bf93e784039123ae542249703e35583e48ea/testing/content/files/en-us/_redirects.txt#L3

So there does be one flaw with it:

```js
"broken_links": [
    {
        "explanation": "Can't resolve /fr/docs/Web/CSS/dumber",
        "id": "link1",
        "href": "/fr/docs/Web/CSS/dumber",
        "suggestion": "/fr/docs/Web/CSS/number",
        "fixable": true,
        "line": 19,
        "column": 16
    }
],
```

Reply:
Thanks, and sorry, I was a bit confused, but now the tests are passing. 👍